editor.planx.uk/src/@planx/components/types.ts

```diff
@@ -14,7 +14,7 @@ export enum TYPES {
   AddressInput = 130,
   ContactInput = 135,
   FileUpload = 140,
-  MultipleFileUpload = 145,
+  FileUploadAndLabel = 145,
```

Comment:
I think this will require a corresponding planx-core change, right?

Reply:
https://github.com/theopensystemslab/planx-core/pull/45 Here we go 😄